pom.xml

```diff
@@ -1,60 +1,76 @@
-  </parent>
-
-  <artifactId>jwt-support</artifactId>
-  <packaging>hpi</packaging>
-
-  <name>JWT Support Plugin</name>
-  <description>Allows generation of a Jenkins-signed JWT to be used as an access token for API calls</description>
-  <url>https://wiki.jenkins-ci.org/display/JENKINS/JWT-Support-Plugin</url>
-  <developers>
-    <developer>
-      <id>agentgonzo</id>
-      <name>Steve Arch</name>
-    </developer>
-    <developer>
-      <id>vivek</id>
-      <name>Vivek Pandey</name>
-        <roles>
-          <role>Original author for blueocean</role>
-        </roles>
-    </developer>
-  </developers>
-
-  <properties>
-      <java.level>8</java.level>
-      <jenkins.version>2.107.2</jenkins.version>
-  </properties>
-
-  <dependencies>
-      <dependency>
-          <groupId>org.bitbucket.b_c</groupId>
-          <artifactId>jose4j</artifactId>
-          <version>0.6.0</version>
-      </dependency>
-      <dependency>
-          <groupId>io.jenkins.blueocean</groupId>
-          <artifactId>blueocean-commons</artifactId>
-          <version>1.5.0</version>
-      </dependency>
-
-      <dependency>
-          <groupId>org.jenkins-ci.plugins</groupId>
-          <artifactId>mailer</artifactId>
-          <version>1.21</version>
-      </dependency>
-
-      <dependency>
-          <groupId>joda-time</groupId>
-          <artifactId>joda-time</artifactId>
-          <version>2.9.9</version>
-      </dependency>
-  </dependencies>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.jenkins-ci.plugins</groupId>
+        <artifactId>plugin</artifactId>
+        <version>3.6</version>
+    </parent>
+
+    <artifactId>jwt-support</artifactId>
+    <version>0.5-SNAPSHOT</version>
+    <packaging>hpi</packaging>
+
+    <name>JWT Support Plugin</name>
+    <description>Allows generation of a Jenkins-signed JWT to be used as an access token for API calls</description>
+    <url>https://wiki.jenkins-ci.org/display/JENKINS/JWT-Support-Plugin</url>
+    <developers>
+        <developer>
+            <id>agentgonzo</id>
+            <name>Steve Arch</name>
+        </developer>
+        <developer>
+            <id>vivek</id>
+            <name>Vivek Pandey</name>
+            <roles>
+                <role>Original author for blueocean</role>
+            </roles>
+        </developer>
+    </developers>
+
+    <scm>
+        <connection>scm:git:ssh://github.com/jenkinsci/jwt-support-plugin.git</connection>
+        <developerConnection>scm:git:ssh://git@github.com/jenkinsci/jwt-support-plugin.git</developerConnection>
+        <url>https://github.com/jenkinsci/jwt-support-plugin</url>
+    </scm>
+
+    <properties>
+        <java.level>8</java.level>
+        <jenkins.version>2.107.2</jenkins.version>
```

Comment:
NIT: normally we don't use LTS versions in plugins, but the weekly that corresponds to the LTS (e.g. 2.107)